src/pages/Admin/BackgroundSelect/BackgroundSelect.tsx

```diff
@@ -13,28 +14,27 @@ import "./BackgroundSelect.scss";
 export interface BackgroundSelectProps {
   venueName: string;
   mapBackground?: string;
+  mapBackgrounds: string[];
+  isLoadingBackgrounds: boolean;
+  error: Error | undefined;
```

Comment:
```suggestion
  error?: Error;
```

Reply:
This is not optional on purpose, I want the TS to cry about this not being passed, I think error should be always passed and handled.

Reply:
You can also view it as It's not the child component job to deal with parent's error. Why even pass error, not least "cry about error not being passed"...